### PR TITLE
Fix RHEL channel names for bootstrapping

### PR DESCRIPTION
**Pull request by witekest**
# Description

Fix RHEL channel names in registering clients instructions.

# Target branches

- [x] master
- [ ] manager-4.3

# Links
- This PR tracks issue uyuni-project/uyuni#7844
- Related development PR uyuni-project/uyuni#8013

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Fix RHEL channel names for bootstrapping
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Fixed RHEL channel names for bootstrapping in Client Configuration Guide
```
We sometimes use the RHEL abbrev here, sometimes the expanded variant.  I do not know whether consistency is needed in this case.

**Review by keichwa — APPROVED** [review]
Thanks for the adjustments!

**Review by keichwa — APPROVED** [review]

**Comment by keichwa** [comment]
@witekest , I think we can also backport it to 4.3?

**Comment by juliogonzalez** [comment]
> @witekest , I think we can also backport it to 4.3? If yes, I can do it for you.

This is only for Uyuni, I don't think you need it in 4.3?

**Comment by juliogonzalez** [comment]
@keichwa why did you close this PR instead of merging it?

**Comment by juliogonzalez** [comment]
I think you need to reopen it and merge it.

**Pull request merged**